app/components/Views/SendFlow/SendTo/index.js

```diff
@@ -347,6 +347,9 @@ class SendFlow extends PureComponent {
 						networkAddressBook[checksummedToSelectedAddress].name) ||
 					(identities[checksummedToSelectedAddress] && identities[checksummedToSelectedAddress].name);
 			} else {
+				toAddressName = await doENSReverseLookup(toSelectedAddress, network);
+
+				toAddressName = toAddressName === undefined ? toSelectedAddress : toAddressName;
```

Comment:
Would this work too?

```suggestion
				toAddressName = toAddressName || toSelectedAddress;
```

Reply:
Yes, tested and implemented! Thank you!